src/libs/PaymentUtils.js

```diff
@@ -7,7 +7,21 @@ import CONST from '../CONST';
 import * as Localize from './Localize';
 
 /**
- * Format the PaymentMethods list
+ * Check to see if user has either a debit card or personal bank account added
+ *
+ * @param {Array} [cardList]
+ * @param {Array} [bankAccountList]
+ * @returns {Boolean}
+ */
+function hasExpensifyPaymentMethod(cardList = [], bankAccountList = []) {
+    return _.some(cardList, card => card) || _.some(bankAccountList, (bankAccountJSON) => {
+        const bankAccount = new BankAccount(bankAccountJSON);
+        return bankAccount.isDefaultCredit();
```

Comment:
Why are we only checking for default credit here? Isn't it any bank account that is verified / open?

Reply:
cc: @marcaaron

Reply:
I might not understand what "default credit" means. Is it not a personal / deposit account?

I thought VBA is not a valid payment method, but maybe wrong about that too. I asked this once [here](https://expensify.slack.com/archives/C01966DBVLL/p1640119438394100?thread_ts=1640119338.393400&cid=C01966DBVLL).